Install_WSL.rst

```diff
@@ -0,0 +1,106 @@
+.. -*- coding: utf-8; indent-tabs-mode:nil; -*-
+
+############################################################
+Spinal Cord Toolbox for Windows : Windows subsyetm for linux
+############################################################
+
+This documentation shows how to install the latest version of SCT. If you want install a different version of SCT please specify the version number you want.
+
+.. contents::
+..
+    1  Windows subsystem for linux 
+    2  environment preparation
+    3  SCT installation 
+    4  FSL installation 
+    5  Usage 
+
+Installation
+************
+
+
+#. Install `Xming <https://sourceforge.net/projects/xming/files/Xming/6.9.0.31/>`_.
+
+#. Install  `Windows subsystem for linux and initialize it <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`_. 
+
+	- Please install the Ubuntu 18.04 LTS distro. 
+
+Environment preparation
+***********************
+
+Run the following command to install various packages that will be needed to install FSL and SCT. This will require your password
+
+.. code-block:: sh
+
+    sudo apt-get -y update
+    sudo apt-get -y install gcc
+    sudo apt-get -y install unzip
+    sudo apt-get install -y python-pip python
+    sudo apt-get install -y psmisc net-tools
+    sudo apt-get install -y git
+    sudo apt-get install -y gfortran
+    sudo apt-get install -y libjpeg-dev
+    echo 'export DISPLAY=127.0.0.1:0.0' >> .profile
+
+
+Install SCT
+*********** 
+
+run the following command to download SCT:
+
+.. code:: sh
+
+    wget https://github.com/neuropoly/spinalcordtoolbox/archive/4.2.1.zip
+
+Once the download is complete run this command to extract SCT from the zip archive:
+
+.. code:: sh
+
+    unzip 4.2.1.zip
+
+Then go inside the folder by running:
+
+.. code:: sh 
+
+    cd spinalcordtoolbox-4.2.1
+
+To install SCT run:
+
+.. code:: sh
+ 
+    yes | ./install_sct
+
+To complete the installation you need to run:
+
+.. code-block:: sh
+
+    cd
+    echo 'export SCT_DIR=/home/sct/sct_dev' >> .profile
```

Comment:
i believe this folder `/home/sct/sct_dev` is not valid based on the code above (`cd spinalcordtoolbox-4.2.1`)

Reply:
ideally we should have SCT installation support `.profile` files. See https://github.com/neuropoly/spinalcordtoolbox/issues/2622.

Reply:
>i believe this folder /home/sct/sct_dev is not valid based on the code above (cd spinalcordtoolbox-4.2.1)

Actually when using ./install_sct it creates a new folder named 'sct' in the 'home' folder that's why I was using it.  At least while using the wget version. 

>ideally we should have SCT installation support .profile files. See neuropoly/spinalcordtoolbox#2622.
 there is a .bashrc actually but I need to add to source it.

Reply:
> Actually when using ./install_sct it creates a new folder named 'sct' in the 'home' folder that's why I was using it. At least while using the wget version. This does not happen with the git version and the Path is then correct but you don't really need to add it as explained below.
> 
> >

Here is what i get when launching the default install from package (4.2.2): (notice the `/Users/julien/sct_4.2.2` folder)
~~~
julien-macbook:~/Desktop/spinalcordtoolbox-4.2.2 $ ./install_sct 


*******************************
* Welcome to SCT installation *
*******************************



Checking OS type and version...

Darwin julien-macbook.local 18.7.0 Darwin Kernel Version 18.7.0: Tue Aug 20 16:57:14 PDT 2019; root:xnu-4903.271.2~2/RELEASE_X86_64 x86_64
ProductVersion:	10.14.6

Checking requirements...


OK!


SCT version ......... 4.2.2
Installation type ... package
Operating system .... osx (14)
Shell config ........ /Users/julien/.bashrc

To improve user experience and fix bugs, the SCT development team is using a
report system to automatically receive crash reports and errors from users.
These reports are anonymous.

Do you agree to help us improve SCT? [y]es/[n]o: y

--> Crash reports will be sent to the SCT development team. Thank you!


SCT will be installed here: [/Users/julien/sct_4.2.2]

Do you agree? [y]es/[n]o: 
~~~

Reply:
note: if we end up using the git clone option, the install will be done locally (no more copied to another folder), which facilitates our task

Reply:
Yes right! 
I think it is better to go with git.